product_docs/docs/tde/15/enabling_tde_epas.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: "Enabling TDE on an existing EDB Postgres Advanced Server database cluster"
+navTitle: Enabling TDE on an existing EPAS cluster
```

Comment:
The reason why I shortened it here is because the title would be three lines long and I felt that would be too long for the navigation. 
 
![Uploading Screenshot 2024-05-29 at 09.42.19.png…]()

Reply:
I would veer towards the full name. If it looks terrible in the menu, so be it, it's a long product name. 

The other option is to create an Enabling TDE subdirectory with an index page and two or more sub pages, with the navtitles being "with an existing EDB..." etc etc